tests/testthat/test-parsnip-survival-standalone.R

```diff
@@ -34,6 +34,8 @@ test_that(".is_censored_right()", {
 })
 
 test_that(".extract_surv_time()", {
+  skip_if_not_installed("parsnip", minimum_version = "1.1.1.9003")
```

Comment:
Nit aka feel free to ignore :)
if you add that skip just above the bits that test the single-row cases, the expectations above it for the multi-row case are still being run.